app/src/main/java/com/randomappsinc/studentpicker/home/HomeActivity.java

```diff
@@ -29,6 +29,8 @@
 public class HomeActivity extends StandardActivity implements
         BottomNavigationView.Listener, CreateListDialog.Listener, PaymentManager.Listener {
 
+    private static final String PREVIOUS_SELECTED_ID = "previousSelectedId";
```

Comment:
This is more descriptive IMHO.
```suggestion
    private static final String PREVIOUSLY_SELECTED_PAGE_ID = "previouslySelectedPageId";
```